projects/arlas-toolkit/src/lib/components/login/login.component.scss

```diff
@@ -111,4 +111,16 @@
   text-decoration: underline;
 }
 
-
+.empty_mode {
```

Comment:
I don't see an empty_mode class in the login component, where is it ?

Reply:
fixed